pycape/function_ref.py

```diff
@@ -49,15 +49,15 @@ class FunctionRef:
     def __init__(
         self,
         id: str,
-        hash: Optional[str] = None,
+        checksum: Optional[str] = None,
         token: Optional[str] = None,
     ):
         id_ = id
-        hash_ = hash
+        checksum_ = checksum
```

Comment:
nit: this variable declaration was only here to make it explicit that we are overriding the Python builtin `hash` function in this scope. since checksum isn't a builtin, we can remove this line and use `checksum` below